ci-operator/step-registry/telco5g/hcp-cluster-setup/telco5g-hcp-cluster-setup-commands.sh

```diff
@@ -125,78 +264,42 @@ cat << EOF > ~/fetch-kubeconfig.yml
     shell: >-
-      oc --kubeconfig=/home/kni/.kube/config_${MGMT_CLUSTER} set data secret/pull-secret -n openshift-config --from-file=.dockerconfigjson=/home/kni/pull-secret.txt
+      oc --kubeconfig=/home/kni/.kube/config_${SNO_NAME} set data secret/pull-secret -n openshift-config --from-file=.dockerconfigjson=/home/kni/pull-secret.txt
 
   - name: Add docker auth to enable pulling containers from CI registry for HCP cluster
     shell: >-
       oc --kubeconfig=/home/kni/.kube/hcp_config_${CLUSTER_NAME} set data secret/pull-secret -n openshift-config --from-file=.dockerconfigjson=/home/kni/pull-secret.txt
 
 EOF
 
-cat << EOF > ~/fetch-information.yml
----
-- name: Fetch information about HCP cluster
-  hosts: hypervisor
-  gather_facts: false
-  tasks:
-
-  - name: Get cluster version
-    shell: oc --kubeconfig=/home/kni/.kube/hcp_config_${CLUSTER_NAME} get clusterversion
-
-  - name: Get cluster operators objects
-    shell: oc --kubeconfig=/home/kni/.kube/hcp_config_${CLUSTER_NAME} get co
-
-  - name: Get nodes
-    shell: oc --kubeconfig=/home/kni/.kube/hcp_config_${CLUSTER_NAME} get node
-
-EOF
-
-# Copy kubeconfig to hypervisor
-echo "Copy kubeconfig to hypervisor"
-ssh -i $SSH_PKEY $COMMON_SSH_ARGS kni@${HYPERV_IP} "kcli scp root@${MGMT_CLUSTER}-installer:/root/ocp/auth/kubeconfig /home/kni/${MGMT_CLUSTER}-kubeconfig"
-
-# Copy automation repo to local SHARED_DIR
-echo "Copy automation repo to local $SHARED_DIR"
-mkdir $SHARED_DIR/repos
-ssh -i $SSH_PKEY $COMMON_SSH_ARGS ${BASTION_USER}@${BASTION_IP} \
-    "tar --exclude='.git' -czf - -C /home/${BASTION_USER} ansible-automation" | tar -xzf - -C $SHARED_DIR/repos/
-
-# Change the host to hypervisor
-echo "Change the host to hypervisor to ${HYPERV_HOST}"
-cd $SHARED_DIR/repos/ansible-automation
-sed -i "s/- hosts: localhost/- hosts: ${HYPERV_HOST}/g" playbooks/hosted_bm_cluster.yaml
-sed -i "s/- hosts: localhost/- hosts: ${HYPERV_HOST}/g" playbooks/remove_bm_cluster.yaml
-
-# Run the playbook to remove any cluster that exists now
-echo "Run the playbook to remove any cluster that exists now"
-ANSIBLE_LOG_PATH=$ARTIFACT_DIR/ansible.log ANSIBLE_STDOUT_CALLBACK=debug ansible-playbook \
-    playbooks/remove_bm_cluster.yaml \
-    -e kubeconfig=/home/kni/${MGMT_CLUSTER}-kubeconfig \
-    -e cluster_name=$CLUSTER_NAME \
-    -e ansible_host=${HYPERV_IP} -e ansible_ssh_user=kni -e ansible_ssh_private_key_file="${SSH_PKEY}" \
-    -e virtual_cluster_deletion=true || true
-
 # Run the playbook to install the cluster
 echo "Run the playbook to install the cluster"
 status=0
+
 ANSIBLE_LOG_PATH=$ARTIFACT_DIR/ansible.log ANSIBLE_STDOUT_CALLBACK=debug ansible-playbook \
-    playbooks/hosted_bm_cluster.yaml \
-    -e kubeconfig=/home/kni/${MGMT_CLUSTER}-kubeconfig \
-    -e pre_step_mgmt_cluster=false \
+    playbooks/sno_hcp_e2e.yml \
     -e hcphost=$CLUSTER_NAME \
-    -e virtual_worker_count=2 \
-    -e hide_sensitive_log=true \
```

Comment:
I think we can retain `hide_sensitive_log=true`

Reply:
ack